tools/ports/sdl2_mixer.py

```diff
@@ -90,7 +95,8 @@ def create(final):
 
 
 def clear(ports, settings, shared):
-  shared.Cache.erase_lib('libSDL2_mixer.a')
+  libname = get_lib_name(settings)
+  shared.Cache.erase_lib(libname)
 
```

Comment:
drive by bug fix...

Reply:
I think you can just do this on one line: `shared.Cache.erase_lib(get_lib_name(settings))` (this would match what other ports seem to do).

What do you think about landing this change (along with the creation of the `get_lib_name` function) as a seperate bugfix since its independent of the version upgrade?

Reply:
Sure, happy to split things up.